HW1/P3/P3.py

```diff
@@ -0,0 +1,16 @@
+import pyspark
+import numpy as np
+from pyspark import SparkContext
+sc = SparkContext()
+
+#read data and sort them
+words = sc.textFile('EOWL_words.txt').map(lambda x: ("".join(np.sort(list(x))),x)).groupByKey().mapValues(list)
```

Comment:
np.sort() time complexity is O(nlogn), I think you can use counting sort or radix sort to make the time complexity to be linear.